packages/devtools/tests/devtools.test.ts

```diff
@@ -48,3 +48,17 @@ test('reloadSession', async () => {
     const newClient = await DevTools.reloadSession(client)
     expect(origSessionId).toBe(newClient)
 })
+
+test('attachSession', async () => {
+    const client = await DevTools.newSession({
+        logLevel: 'trace',
+        capabilities: {
+            browserName: 'chrome'
+        }
+    })
+    const clientAttach = await DevTools.attachToSession({
+        sessionId: client.sessionId,
+        capabilities: { browserName: 'chrome' }
+    })
+    expect(JSON.stringify(clientAttach)).toEqual(JSON.stringify(client))
```

Comment:
Not sure if this a valid case to test this feature

Reply:
Not really. For unit tests it is more important that you check if certain methods has been called with the expected parameters. This is more an integration test that has not much expressiveness and is prone for errors.